pkg/controller/plan/migration.go

```diff
@@ -1647,12 +1647,14 @@ func (r *Migration) updateConversionProgress(vm *plan.VMStatus, step *plan.Step)
 			break
 		}
 
-		el9, err := r.Context.Plan.VSphereUsesEl9VirtV2v()
+		coldLocal, err := r.Context.Plan.VSphereColdLocal()
 		switch {
 		case err != nil:
 			return liberr.Wrap(err)
-		case el9:
-			if err := r.updateConversionProgressEl9(pod, step); err != nil {
+		case coldLocal:
+			// FIXME: will this work for in-place conversions now that it also uses
+			// virt-v2v-monitor when executing virt-v2v-in-place?
```

Comment:
Nope different flow, as we get the progress from CNV CDI instead of monitoring the v2v output. 
I'll remove the comment